src/display/canvas.js

```diff
@@ -3285,6 +3179,10 @@ class CanvasGraphics {
     }
 
     ctx.scale(scaleX, scaleY);
+    const newPath = new Path2D();
+    SCALE_MATRIX[0] = 1 / scaleX;
+    SCALE_MATRIX[3] = 1 / scaleY;
```

Comment:
Could this explain the new test failures, since I don't believe that you can change a `DOMMatrix`-instance that way?

Maybe this would work?
```suggestion
    SCALE_MATRIX.a = 1 / scaleX;
    SCALE_MATRIX.d = 1 / scaleY;
```